ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 5.5.0f3
+m_EditorVersion: 5.5.1f1
```

Comment:
Maybe throw a more informative assert?

Lest other wayward devs meet the same fate...